src/components/Feature.js

```diff
@@ -5,18 +5,37 @@ import FlagContext from '../utils/FlagContext';
 import Config from '../config';
 
 export class Feature extends Component {
+  componentDidMount() {
+    if (this.props.flags.isLDReady && this.props.onReady) {
```

Comment:
could make sure it's callable, maybe overkill

Reply:
I feel like that's an implementers concern

Reply:
It may be even more confusing if you accidentally send in an uncallable method and nothing happens because we did an internal check and threw it away